packages/node_modules/@webex/plugin-meetings/src/config.js

```diff
@@ -26,7 +26,7 @@ export default {
     },
     stats: {
       // Enable the webrtc stats analyzer that emits quality degradation events
-      enableStatsAnalyzer: false,
+      enableStatsAnalyzer: true,
```

Comment:
Have we confirmed that this feature is working properly as expected with Varun and team?

Reply:
good point i though the rebase would over ride it let me change it back